src/main/models/myCaseWorkModel.ts

```diff
@@ -28,7 +53,12 @@ export class MyCaseWorkModel {
 
     tasks.push(task0);
     tasks.push(task2);
-    tasks.push(task3);
+    tasks.push(task10);
```

Comment:
We can only initialise this object with Taylor House tasks because it's the default filter.